emulators/retroarch/Portfile

```
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           xcode 1.0
+
+github.setup        libretro RetroArch 1.9.1 v
+revision            0
+
+name                retroarch
+platforms           darwin
+license             GPL-3
+maintainers         {gmail.com:audvare @Tatsh} openmaintainer
+categories          emulators games
+description         Frontend for the libretro API.
+long_description    ${description}
+
+checksums           rmd160  2c4f65be5cb1be55da9f242af9721c5098fe01dc \
+                    sha256  e1fb48a0c9f8ce4ca31699c9611727e3b04b127d7c7eb2de171809279681ddf3 \
+                    size    39878651
+
+patchfiles          patch-${name}-library-dirs.diff
+
+xcode.project       ${worksrcpath}/pkg/apple/RetroArch.xcodeproj
+xcode.target        RetroArch
+xcode.destroot.settings \
+    CODE_SIGN_IDENTITY=- \
+    CODE_SIGN_STYLE=Manual \
+    ENABLE_HARDENED_RUNTIME=NO \
+    GCC_TREAT_WARNINGS_AS_ERRORS=NO
+
+# Compile and install in one step, for now. See https://trac.macports.org/ticket/57137
+build {}
+
+if {${os.major} < 10} {
+    known_fail yes
+    pre-fetch {
+        ui_error "${name} @${version} requires OS X 10.6 or later."
+        return -code error "incompatible OS X version"
```

10.7 and earlier are "Mac OS X" not "OS X".
```suggestion
        ui_error "${name} @${version} requires Mac OS X 10.6 or later."
        return -code error "incompatible Mac OS X version"
```